libs/domain-service/src/__tests__/functionnal/domain-service.react.test.tsx

```diff
@@ -77,6 +77,7 @@ describe("useNamingService", () => {
         address: "forced mocked address",
```

Comment:
Not related to your changes but I think there is a typo on the folder, it's "functional" with a single n :D